test/runtests.jl

```diff
@@ -28,7 +28,8 @@ tests = [
     "gradlogpdf",
     "truncate",
     "truncatednormal",
-    "generalizedextremevalue"]
+    "generalizedextremevalue",
+    "largenbetabinomialpdf"]
```

Comment:
There are existing tests, they're just part of a larger, less obvious framework. See https://github.com/JuliaStats/Distributions.jl/blob/master/test/discrete_hand_coded_test.json.